src/algebraic_topology/dold_kan/equivalence_additive.lean

```diff
@@ -0,0 +1,62 @@
+/-
+Copyright (c) 2022 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou
+-/
+
+import algebraic_topology.dold_kan.n_comp_gamma
+
+/-! The Dold-Kan equivalence for additive categories.
+
+This file defines `preadditive.dold_kan.equivalence` which is the equivalence
+of categories `karoubi (simplicial_object C) ≌ karoubi (chain_complex C ℕ)`.
+
+-/
+
+noncomputable theory
+
+open category_theory category_theory.category category_theory.limits
+  category_theory.idempotents algebraic_topology.dold_kan
+
+variables {C : Type*} [category C] [preadditive C]
+
+namespace category_theory
+
+namespace preadditive
+
+namespace dold_kan
+
+/-- The functor `karoubi (simplicial_object C) ⥤ karoubi (chain_complex C ℕ)` of
```

Comment:
Can you please expand these docstrings with more detail for users that don't know all the maths by heart?

More generally: when using Dold-Kan, should we rely on good defeqs, or will you provide an API for using it?

Reply:
With this PR, we get `karoubi (simplicial_object C) ≌ karoubi (chain_complex C ℕ)`, from which one may get `simplicial_object C ≌ chain_complex C ℕ` when `C` is idempotent complete (e.g. `C` is abelian). In future PRs, these equivalences `simplicial_object C ≌ chain_complex C ℕ` are obtained with the best possible defeqs. For example, when `C` is abelian, the equivalence `abelian.dold_kan.equivalence` will be such that the `functor` of the equivalence is defeq to `algebraic_topology.normalized_Moore_complex` (and not a weird composition of some functors and some quasi-inverses) and the `inverse` functor will be `Γ₀`. I already have the code, but I think that this could wait for the post-mathlib4-port era.
Then, what I will do advertise this as `TODO` in this `equivalence_additive` file, clarify some docstrings, and update the internal references in the intros of the files in this directory.